pages/en/developing/assemblyscript-api.mdx

```diff
@@ -444,19 +444,20 @@ The code generated by `graph codegen` also includes classes for the smart contra
 A common pattern is to access the contract from which an event originates. This is achieved with the following code:
 
 ```typescript
-// Import the generated contract class
-import { ERC20Contract } from '../generated/ERC20Contract/ERC20Contract'
+// Import the generated contract class and generated Transfer event class
+import { ERC20Contract, Transfer as TransferEvent } from '../generated/ERC20Contract/ERC20Contract'
 // Import the generated entity class
 import { Transfer } from '../generated/schema'
```

Comment:
I think this line can actually be deleted for the purpose of this snippet?

Reply:
Line 450?